### PR TITLE
OCPQE-26445: Add external OIDC jobs for Hypershift AKS

### PR DESCRIPTION
**Pull request by fxierh**
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58012/rehearse-58012-periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-ext-oidc-guest-f7/1848453158603329536

https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58012/rehearse-58012-periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-full-cert-guest-f14/1848453158813044736

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/58012/rehearse-58012-pull-ci-openshift-hypershift-main-e2e-azure-aks-ovn-conformance/1848453158548803584 (cluster creation succeeded, test case failure unrelated to this change)

### CODE REVIEW

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-ext-oidc-guest-f7

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-ext-oidc-guest-f7

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-ext-oidc-guest-f7

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-ext-oidc-guest-f7

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-ext-oidc-guest-f7

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-ext-oidc-guest-f7

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-ext-oidc-guest-f7

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-ext-oidc-guest-f7

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-ext-oidc-guest-f7 pull-ci-openshift-hypershift-main-e2e-azure-aks-ovn-conformance pull-ci-openshift-hypershift-main-e2e-aks periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-full-cert-guest-f14

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-ext-oidc-guest-f7 pull-ci-openshift-hypershift-main-e2e-azure-aks-ovn-conformance pull-ci-openshift-hypershift-main-e2e-aks periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-full-cert-guest-f14

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml [hunk 1]
```diff
@@ -1505,6 +1505,14 @@ tests:
...
     - chain: openshift-e2e-test-hypershift-qe
     workflow: cucushift-installer-rehearse-azure-aks-hypershift-etcd-disk-encryption-guest
+- as: azure-aks-hypershift-ext-oidc-guest-f7
+  cron: 23 19 5,12,19,27 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~NonPreRelease&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;~Serial&;~Disruptive&
+      TEST_TIMEOUT: "30"
+    workflow: cucushift-installer-rehearse-azure-aks-hypershift-ext-oidc-guest
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml:1515]: I would suggest to add/override `test` there, though the test is already defined in the workflow. Teammates generally comes to this folder to check config things.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml:1515]: You can add a new test chain if there are multiple test steps included and you don't want to copy/paste them for several places/versions.

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-aks-hypershift-ext-oidc-guest-f7

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml (hunk 1 above)
Comment (fxierh) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml:1514]: Updated.

**Pull request merged**